src/assets/goodsData.ts

```diff
@@ -0,0 +1,32 @@
+export const GOODS_DATA: Goods[] = [
+  {
+    id: 123,
+    name: 'BBQ 양념치킨+크림치즈볼+콜라1.25L',
+    imageURL: 'https://st.kakaocdn.net/product/gift/product/20231030175450_53e90ee9708f45ffa45b3f7b4bc01c7c.jpg',
+    price: {
+      basicPrice: 29000,
```

Comment:
큰 숫자를 사용할 때는 numeric separtor를 활용해서 가독성을 높여보시면 좋을거 같아요

```suggestion
      basicPrice: 29_000,
```